index.js

```diff
@@ -215,10 +217,8 @@ function init (ssb, config) {
     },
     findByFeedId: scuttle.link.findGroupByFeedId,
     application: {
-      create: scuttle.application.create,
-      list: scuttle.application.list,
-      get: scuttle.application.get,
-      accept: scuttle.application.accept
+      ...scuttle.application,
+      get: scuttle.application.read // legacy
```

Comment:
ahh okay so you changed application.get to application.read, and putting this here so its still usable and doesnt break old usage

Reply:
I ended up deciding to leave it as get. Need to push that up.
It's get in my mind because this is more than a crut read